Tests/PublisherTests/Mocks/MockAblyPublisherService.swift

```diff
@@ -38,12 +38,15 @@ class MockAblyPublisherService: AblyPublisherService {
     var sendEnhancedAssetLocationUpdateParamLocationUpdate: EnhancedLocationUpdate?
     var sendEnhancedAssetLocationUpdateParamTrackable: Trackable?
     var sendEnhancedAssetLocationUpdateParamCompletion: ResultHandler<Void>?
+    var sendEnhancedAssetLocationUpdateParamCompletionHandler: ((ResultHandler<Void>?) -> Void)?
     func sendEnhancedAssetLocationUpdate(locationUpdate: EnhancedLocationUpdate, forTrackable trackable: Trackable, completion: ResultHandler<Void>?) {
         sendEnhancedAssetLocationUpdateCounter += 1
         sendEnhancedAssetLocationUpdateCalled = true
         sendEnhancedAssetLocationUpdateParamLocationUpdate = locationUpdate
         sendEnhancedAssetLocationUpdateParamTrackable = trackable
         sendEnhancedAssetLocationUpdateParamCompletion = completion
+        
```

Comment:
It looks like an unnecessary newline, maybe let's remove it? :wink:

Reply:
2e7c30a2d6e2aaad90c38ced5a0842537e0aa4dd